nautobot/circuits/api/serializers.py

```diff
@@ -17,8 +17,11 @@
     TaggedObjectSerializer,
 )
 from nautobot.tenancy.api.nested_serializers import NestedTenantSerializer
-from .nested_serializers import *
+from .nested_serializers import NestedCircuitSerializer, NestedCircuitTypeSerializer, NestedProviderSerializer
 
+# This import is separated from the above import, as the variable(s) are not actually used anywhere in this file,
+# but still required for brief fields functionality to work
+from .nested_serializers import NestedCircuitTerminationSerializer  # noqa: F401
```

Comment:
I don't think there's a great need to keep them separated in this way, I'd prefer just:

```suggestion
# Nested serializers must be imported for "brief" fields functionality to work.
from .nested_serializers import (  # noqa: F401
    NestedCircuitSerializer,
    NestedCircuitTerminationSerializer,
    NestedCircuitTypeSerializer,
    NestedProviderSerializer,
)
```

Same comment on each other `serializers.py` file.